src/Month.js

```diff
@@ -37,7 +37,7 @@ class MonthView extends React.Component {
 
   componentWillReceiveProps({ date }) {
     this.setState({
-      needLimitMeasure: !dates.eq(date, this.props.date, 'month'),
+      needLimitMeasure: true,
```

Comment:
this not correct, now any component update will trigger a measure, which is expensive.